### DISCUSSION TITLE
is there some client code example to show how to interact with seaweedfs?

### DISCUSSION THREAD

**Discussion by stardustman** [seaweedfs/seaweedfs#2480] [Q&A] [answered]
need some example code to demonstrate how to interact with seweedfs.

**Comment by chrislusf** [comment, marked as answer]
not clear what you want. 
Simple http request can "interact" with SeaweedFS already.

**Reply by stardustman** [reply]
can I use the `aws-sdk-go` to write golang code to  interact with SeaseedFS directly?
